pages/docs/administration/getting-started/configuration/app-configuration.mdx

```diff
@@ -0,0 +1,138 @@
+---
+priority: 100100
+---
+
+import DocsWrapper from '@/src/docs/DocsWrapper'
+
+# App Configuration
+
+export default ({ children }) => <DocsWrapper>{children}</DocsWrapper>
+
+## Note
+
+Below listed configurations can be changed during dockerbuild.
+
+- default application configuration is applied from config/default.js.
```

Comment:
These are many more, it's odd to pick out just these few options.

Reply:
I'd argue these are either obvious or should be documented within the config themselves.  Not here.

Reply:
added link to node config docs page and reference to helm making changes to application during runtime.